pkg/components/kubernetes_loader.go

```diff
@@ -63,10 +65,11 @@ func requestControlPlane(url string) ([]byte, error) {
 	}
 	err := client.Do(req, resp)
 	if err != nil {
-		// Request failed, retry again
+		// Request failed, try again
+		fmt.Println("trying again");
```

Comment:
Can you import `log "github.com/Sirupsen/logrus"` and replace this with `log.Info("retrying getting components")`?

Reply:
Whoops, didn't mean to keep that in there. All good now